example/config.yaml

```diff
@@ -11,4 +11,23 @@ gateways:
         ssl_enabled: true
         use_public_ip: true
         health_path_override: "/health/liveness"
-
+        add_rewrite_rule: true
```

Comment:
want to include all example changes? 
certificates & if any additional

Reply:
Added new settings and corrected position of some existing settings.